packages/main/src/Menu.hbs

```diff
@@ -0,0 +1,100 @@
+<ui5-responsive-popover
+	id="{{_id}}-menu-rp"
+	class="ui5-menu-rp"
+	dir={{effectiveDir}}
+	horizontal-align="Left"
+	placement-type={{placementType}}
+	vertical-align={{verticalAlign}}
+	hide-arrow
+	allow-target-overlap
+	?sub-menu={{_isSubMenu}}
+	@before-close={{_beforePopoverClose}}
+>
+{{#if isPhone}}
+	<div
+		slot="header"
+		class="ui5-menu-dialog-header"
+	>
+	{{#if isSubMenuOpened}}
+		<ui5-button
+			icon="nav-back"
```

Comment:
All the icons in the template should be imported explicitly, the test pages do not fail because all the icons are imported, bundled and available, but in real scenario an application will import only the icons it uses and those used internally should be imported by the components (in this case the menu) that use them. There are multiple examples, f.e. check the CheckBox that imports its checkmark icon.

Reply:
Done